CHANGELOG.md

```diff
@@ -0,0 +1,206 @@
+
+**Minor Changes:**
+* Add cilium_lb_act BPF map with counters of opened and closed connections (cilium/cilium#32584, @AwesomePatrol)
+* Adds "aws-enable-ipv6-prefix-delegation" operator option for configuring AWS ENI IPv6 prefix delegation. (cilium/cilium#31145, @danehans)
+* Allow configuring RAM-backed clustermesh-apiserver's etcd storage for improved performance in high-scale/high-churn environments (cilium/cilium#32823, @giorio94)
+* bpf: allow policy verdict notifications in bpf_host (cilium/cilium#32934, @jibi)
+* Change default Clustermesh control plane upgrade strategy to use surge strategy (cilium/cilium#32999, @marseel)
+* chore: Bump spire agent and server versions (cilium/cilium#33136, @sayboras)
+* Cilium now supports Kubernetes Service TrafficDistribution. To access this feature, use `--enable-service-topology` when running Cilium. (cilium/cilium#32678, @robscott)
+* datapath: Add support for skipping direct routes on different L2 networks (cilium/cilium#32733, @jleeh)
+* docs: Deprecate support for podnetwork etcd (cilium/cilium#33030, @joestringer)
+* envoy: Bump envoy version to v1.29.5 (cilium/cilium#32915, @sayboras)
+* etcd, clustermesh: generalize and untangle the custom dialer logic for automatic DNS name to service ClusterIP translation (cilium/cilium#32916, @giorio94)
+* externalTrafficPolicy support for Cilium Ingress and GatewayAPI (cilium/cilium#32873, @PhilipSchmid)
+* Formally define and validate the cluster name format (cilium/cilium#32641, @giorio94)
+* gateway-api: Bump to version v1.1.0 (cilium/cilium#32233, @sayboras)
+* helm: loadBalancerClass for Cluster Mesh APIserver (cilium/cilium#33033, @PhilipSchmid)
+* hubble: node labels (cilium/cilium#32851, @kaworu)
+* ingress: Support headless service (cilium/cilium#32644, @sayboras)
+* Introduce --force-device-detection option to apply the auto-detection criteria also when devices are explicitly listed with --devices. (cilium/cilium#32730, @kvaps)
+* Introduce granular etcd permissions to access KVstoreMesh cached data (cilium/cilium#33082, @giorio94)
+* More validation has been added to the CiliumNetworkPolicy and CiliumClusterwideNetworkPolicy CRDs. Policies that may have been ignored by the Cilium agent will now be rejected by the Kubernetes API server. (cilium/cilium#32814, @squeed)
+* Remove etcd.managed Helm setting (cilium/cilium#32921, @joestringer)
+* Support Egress Gateway for endpoints that are also selected by a L7 Network Policy. (cilium/cilium#32828, @ysksuzuki)
+* Switch the RBAC used for hubble certificate generation in `cronJob` mode to namespace-scoped. (cilium/cilium#33027, @giorio94)
+* ui: v0.13.1 release (cilium/cilium#32852, @geakstr)
+* When upgrading, users can experience a change to their configuration if they were overriding the `k8s-heartbeat-timeout` flag.
 K8s client timeout and keep alive are no longer getting values from the `k8s-heartbeat-timeout` flag, but have default values (30 seconds). (cilium/cilium#32625, @dlapcevic)
```

Comment:
Nit: Could we remove this `^M` character here?

Reply:
Fixed